esp/src/eclwatch/nls/hpcc.js

```diff
@@ -248,10 +255,14 @@ define({root:
     Largest: "Largest",
     LargestFile: "Largest File",
     LargestSize: "Largest Size",
+    LastMessage: "Last Message",
+    LastMessage2: "Last Message:",
```

Comment:
How about in the html I can do something like `<label class="Prompt" for="${id}LastMessage">${i18n.LastMessage2}:</label>`